worlds/raft/items.json

```diff
@@ -7,335 +7,460 @@
     {
         "id": 47002,
         "progression": false,
-        "name": "Leather helmet"
+        "name": "Big backpack"
     },
     {
         "id": 47003,
         "progression": false,
-        "name": "Leather body armor"
+        "name": "Leather helmet"
     },
     {
         "id": 47004,
         "progression": false,
-        "name": "Leather greaves"
+        "name": "Leather body armor"
     },
     {
         "id": 47005,
         "progression": false,
-        "name": "Flippers"
+        "name": "Leather greaves"
     },
     {
         "id": 47006,
         "progression": false,
-        "name": "Head light"
+        "name": "Flippers"
     },
     {
         "id": 47007,
         "progression": false,
-        "name": "Oxygen bottle"
+        "name": "Head light"
     },
     {
         "id": 47008,
-        "progression": true,
-        "name": "Zipline tool"
+        "progression": false,
+        "name": "Advanced head light"
     },
     {
         "id": 47009,
-        "progression": true,
-        "name": "Empty bottle"
+        "progression": false,
+        "name": "Oxygen bottle"
     },
     {
         "id": 47010,
-        "progression": false,
-        "name": "Clay bowl"
+        "progression": true,
+        "name": "Zipline tool"
     },
     {
         "id": 47011,
         "progression": false,
-        "name": "Bucket"
+        "name": "Electric zipline tool"
     },
     {
         "id": 47012,
         "progression": false,
-        "name": "Healing salve"
+        "name": "Empty bottle"
```

Comment:
As mentioned on discord, I believe this should be progression as `raft_can_craft_plasticBottle` requires this item

Reply:
Good catch again :)